it-IT.json

```diff
@@ -1,181 +1,181 @@
 {
   "1 {0} = {1} {2}": "1 {0} = {1} {2}",
-  "<span>Please make sure you're visiting</span><a>app.minswap.org</a><span>- bookmark it to be safe.</span>": "<span>Please make sure you're visiting</span><a>app.minswap.org</a><span>- bookmark it to be safe.</span>",
-  "A collateral is required in order to interact with smart contracts on Cardano. You can select an UTxO entry with a value between 5 and 20 as a collateral.": "A collateral is required in order to interact with smart contracts on Cardano. You can select an UTxO entry with a value between 5 and 20 as a collateral.",
+  "<span>Please make sure you're visiting</span><a>app.minswap.org</a><span>- bookmark it to be safe.</span>": "<span>Per favore assicurati di essere su </span><a>app.minswap.org</a><span>- per sicurezza salvalo fra i preferiti.</span>",
+  "A collateral is required in order to interact with smart contracts on Cardano. You can select an UTxO entry with a value between 5 and 20 as a collateral.": "Un collaterale è nesessario per interagire con gli smart-contracts su Cardano. Puoi selezionare un UTxO come collaterale con un valore compreso fra 5 e 20.",
   "APR": "APR",
-  "Action": "Action",
-  "Add liquidity": "Add liquidity",
-  "Adding liquidity of {1} {2} and {3} {4}": "Adding liquidity of {1} {2} and {3} {4}",
-  "All": "All",
+  "Action": "Azione",
+  "Add liquidity": "Aggiungi liquidità",
+  "Adding liquidity of {1} {2} and {3} {4}": "Aggiungendo liquidità di {1} {2} e {3} {4}",
+  "All": "Tutti",
   "Audio": "Audio",
-  "Balance": "Balance",
-  "Batcher Fee": "Batcher Fee",
-  "Browse your pending/past orders.": "Browse your pending/past orders.",
-  "Cancel": "Cancel",
-  "Canceled order": "Canceled order",
-  "Canceling order": "Canceling order",
-  "Cancelled": "Cancelled",
+  "Balance": "Saldo",
+  "Batcher Fee": "Tassa di batcher",
+  "Browse your pending/past orders.": "Visualizza i tuoi ordini passati/in sospeso",
+  "Cancel": "Cancellla",
+  "Canceled order": "Ordine cancellato",
+  "Canceling order": "Cancellando l'ordine",
```

Comment:
non sarebbe meglio "ordine in eliminazione" non è la traduzione letterale, ma credo abbia più senso